src/NServiceBus.Testing/Saga.cs

```diff
@@ -345,7 +345,7 @@ public Saga<T> AssertSagaCompletionIs(bool complete)
         /// </summary>
         public Saga<T> ExpectTimeoutToBeSetIn<TMessage>(Func<TMessage, TimeSpan, bool> check = null)
         {
-            testableMessageHandlerContext.ExpectedInvocations.Add(new ExpectDelayDeliveryWith<TMessage>(check));
+            testContext.ExpectedInvocations.Add(new ExpectDelayDeliveryWith<TMessage>(check));
```

Comment:
True. But it is less trainwreck and since the major responsibility of that class is capturing the expectations it would be closer to that responsibility. But I have no really strong arguments.